internal/credential/vault/private_library.go

```diff
@@ -549,6 +548,7 @@ func (r *Repository) getIssueCredLibraries(ctx context.Context, requests []crede
 			return nil, errors.Wrap(ctx, err, op)
 		}
 		decryptedLibs = append(decryptedLibs, pl.toTypedIssuingCredentialLibrary())
+
```

Comment:
Remove this. I recommend looking through your own PR for unintentional changes like this to save us both time in review.